### ISSUE TITLE
[HW] Manoj Rattihalli

### ISSUE THREAD

**Issue by manojrattihalli** [datastaxdevs/workshop-introduction-to-nosql#535]
**Name:** Manoj Rattihalli
**Email:** manojrattihalli@gmail.com
**Linkedin Profile:** <LINK>

Attach the homework screenshots below:
-----------------------------------------
![image](https://user-images.githubusercontent.com/89215613/130115630-6b9b6e25-7619-465f-8811-7be5585e5240.png)
![image](https://user-images.githubusercontent.com/89215613/130115850-2000c23c-80c4-47b8-b9d5-d0bf95a65057.png)
![image](https://user-images.githubusercontent.com/89215613/130115940-7c371f86-5580-486e-a0e2-367cbf0096b2.png)
![image](https://user-images.githubusercontent.com/89215613/130116015-21abad73-05b7-4fb4-a118-c2b6a02e5ab5.png)
![image](https://user-images.githubusercontent.com/89215613/130116129-df23aedf-4edf-45a9-85e3-4726813e65fe.png)

Swagger screenshots:
![image](https://user-images.githubusercontent.com/89215613/130116787-89e4880d-d2f0-46a2-b118-1810e7d83437.png)
![image](https://user-images.githubusercontent.com/89215613/130116928-497ac707-0071-48d8-b296-2af232a4f947.png)
![image](https://user-images.githubusercontent.com/89215613/130117000-75459fdf-4c4e-43f3-9b27-c8d50fe25a5e.png)
![image](https://user-images.githubusercontent.com/89215613/130117363-592dbb3b-bc8d-4d21-8037-61866ea30883.png)

3f Retrieve a document from its id :
![image](https://user-images.githubusercontent.com/89215613/130117829-8bcc593d-21aa-440e-8053-256cd1a17603.png)

3g Search document from a where clause :
![image](https://user-images.githubusercontent.com/89215613/130118322-cbc3c69f-37be-4b1d-b00c-b4eebe2a76cd.png)

4b:
![image](https://user-images.githubusercontent.com/89215613/130119460-be5e197f-b652-4f33-935b-ff7e95d5f243.png)
![image](https://user-images.githubusercontent.com/89215613/130119704-84ba9685-183f-40aa-a909-0578add1c371.png)
![image](https://user-images.githubusercontent.com/89215613/130119905-690b5fb0-6918-4fea-a98c-785ffe9a4342.png)
![image](https://user-images.githubusercontent.com/89215613/130120029-c7b000ec-dbed-4270-a789-7dbab5a0d864.png)
![image](https://user-images.githubusercontent.com/89215613/130120133-c0106f13-709a-4281-bb63-ad5c766b6cec.png)

**Comment by SonicDMG** [comment]
Hi there Manoj, thank you for your submission. I see you got through the workshop content, awesome. We are missing step 2 from the homework though.
2. Complete [try-it-out scenario](https://www.datastax.com/try-it-out) and make a screenshot of the "scenario completed" screen

Please complete this and provide a screenshot of the completed scenario.

**Comment by SonicDMG** [comment]
Also, I had to remove many of your images because some included your Astra token. Please delete your token from the AstraUI to ensure there are no security issues.

**Comment by manojrattihalli** [comment]
Hi David,

Since I had already completed the exercises, I did a 'describe' of keyspace
and shared the screen shot. I will redo the 2.x exercises and share
screenshots again by Monday. Hope the databases I created will be active
(please confirm).

Thanks
Manoj Rattihalli

**Comment by SonicDMG** [comment]
Where those "describes" from the katacoda scenario outside of Astra DB? I apologize if I removed your completion evidence, but I didn't think I saw the screens from the actual scenario. If you said you did them I will take that.

**Comment by manojrattihalli** [comment]
Hi David

The 'describe' was from Astra DB (nosql1) and I had covered the exercises
that were discussed during the 2hrs training.

For homework, is there a different exercise/question that needs to be
solved? Apologies if I have missed it. Please share the path to HW exercise
questions if it is a different one. I will add relevant screenshots upon
your confirmation.

Thanks
Manoj Rattihalli

**Comment by SonicDMG** [comment]
Yes, for the homework there is this scenario as well https://www.datastax.com/try-it-out

Shouldn't take very long to do. :)
